spec/requests/works_spec.rb

```diff
@@ -150,214 +150,258 @@
-          work_type: 'text',
-          contact_email: 'io@io.io',
-          abstract: 'test abstract',
-          keywords_attributes: {
-            '0' => { '_destroy' => 'false', 'label' => 'Feminism', 'uri' => 'http://id.worldcat.org/fast/922671' }
-          },
-          license: 'CC0-1.0',
-          release: 'immediate'
-        }
-      end
+      context 'with a minimal set' do
+        let(:collection) { create(:collection, depositors: [user]) }
+        let(:work_params) do
+          {
+            title: 'Test title',
+            work_type: 'text',
+            contact_email: 'io@io.io',
+            abstract: 'test abstract',
+            keywords_attributes: {
+              '0' => { '_destroy' => 'false', 'label' => 'Feminism', 'uri' => 'http://id.worldcat.org/fast/922671' }
+            },
+            license: 'CC0-1.0',
+            release: 'immediate'
+          }
+        end
 
-      it 'displays the work' do
-        post "/collections/#{collection.id}/works", params: { work: work_params }
-        expect(response).to have_http_status(:found)
-        work = Work.last
-        expect(work.contributors).to be_empty
-        expect(work.attached_files).to be_empty
-        expect(work.keywords.size).to eq 1
-        expect(work.published_edtf).to be_nil
-        expect(work.created_edtf).to be_nil
-        expect(work.embargo_date).to be_nil
-        expect(work.subtype).to be_empty
+        it 'displays the work' do
+          post "/collections/#{collection.id}/works", params: { work: work_params }
+          expect(response).to have_http_status(:found)
+          work = Work.last
+          expect(work.contributors).to be_empty
+          expect(work.attached_files).to be_empty
+          expect(work.keywords.size).to eq 1
+          expect(work.published_edtf).to be_nil
+          expect(work.created_edtf).to be_nil
+          expect(work.embargo_date).to be_nil
+          expect(work.subtype).to be_empty
+          expect(work.state).to eq 'first_draft'
+          expect(DepositJob).not_to have_received(:perform_later)
+        end
       end
-    end
 
-    describe 'update work with an attachment' do
-      let(:work) { create(:work, :with_attached_file) }
-      let(:user) { work.depositor }
-      let(:work_params) do
-        {
-          title: 'New title',
-          work_type: 'text',
-          contact_email: 'io@io.io',
-          abstract: 'test abstract',
-          attached_files_attributes: {
-            '0' => { 'label' => 'two', '_destroy' => '', 'hide' => '0', 'id' => work.attached_files.first.id }
-          },
-          keywords_attributes: {
-            '0' => { '_destroy' => 'false', 'label' => 'Feminism', 'uri' => 'http://id.worldcat.org/fast/922671' }
-          },
-          license: 'CC0-1.0',
-          release: 'immediate'
-        }
-      end
+      context 'with a moderated collection' do
+        let(:collection) { create(:collection, :with_reviewers, depositors: [user]) }
```

Comment:
is it the "with_reviewers" that makes this test relevant to the code changes?

Reply:
Yes.